aws-ssm-patchbaseline/aws-ssm-patchbaseline.json

```diff
@@ -280,14 +280,19 @@
         "Name"
     ],
     "createOnlyProperties": [
-        "/properties/OperatingSystem"
+        "/properties/OperatingSystem",
+        "/properties/DefaultBaseline",
+        "/properties/Tags"
     ],
     "primaryIdentifier": [
         "/properties/Id"
     ],
     "readOnlyProperties": [
         "/properties/Id"
     ],
+    "writeOnlyProperties": [
```

Comment:
what are writeonly properties for?

Reply:
writeOnlyProperties
Resource properties that can be specified by the user, but cannot be returned by a read or list request. Write-only properties are often used to contain passwords, secrets, or other sensitive data.